.github/workflows/ecosystem-ci-from-pr.yml

```diff
@@ -115,6 +118,7 @@ jobs:
           pnpm tsx ecosystem-ci.ts
           --branch ${{ inputs.branchName }}
           --repo ${{ inputs.repo }}
+          --commit ${{ inputs.commit }}
```

Comment:
We need to pass the same argument to the `pnpm tsx ecosystem-ci.ts` in the `execute-all` job below as well.

Reply:
done!